tests/inspectdb/tests.py

```diff
@@ -74,11 +74,9 @@ def test_field_types(self):
             assertFieldType('url_field', "models.CharField(max_length=200)")
         assertFieldType('date_field', "models.DateField()")
         assertFieldType('date_time_field', "models.DateTimeField()")
-        if connection.features.can_introspect_ip_address_field:
-            assertFieldType('gen_ip_address_field', "models.GenericIPAddressField()")
-        elif not connection.features.interprets_empty_strings_as_nulls:
+        if not connection.features.interprets_empty_strings_as_nulls:
             assertFieldType('gen_ip_address_field', "models.CharField(max_length=39)")
-        if connection.features.can_introspect_time_field:
+        if connection.features.introspected_field_types['TimeField'] == 'TimeField':
```

Comment:
Adding a local variable like `introspected_field_types = connection.features.introspected_field_types` in some tests could make things nicer so that many assertions don't have to be multi-lined.